app/controllers/api/account.php

```diff
@@ -780,7 +780,7 @@
     });
 
 App::put('/v1/account/sessions/magic-url')
-    ->desc('Create Magic URL session (confirmation)')
+    ->desc('Create magic URL session (confirmation)')
```

Comment:
Not related to this PR, @Dksie09 please ignore message below.

What do y'all think about just calling this `Confirm magic URL session`. Like why did we use brackets?